### PR TITLE
[FEATURE] Create offer frontend CRUD

### PR DESCRIPTION
**Pull request by Ando22**
## Description
creating a frontend offer feature for users to create offer requests for an Item.

this PR will be merged when this [issue](https://github.com/idurar/erp-crm/issues/389) is fixed 

## Related Issues
This PR is related to this [issue](https://github.com/idurar/erp-crm/issues/390). 

## Checklist

- [ ] I have tested these changes
- [ ] I have made corresponding changes to the codebase

### CODE REVIEW

File: frontend/.env
Comment (salahlalami) [inline frontend/.env:1]: add .env to .gitignore

**Review by salahlalami — CHANGES_REQUESTED** [review]

File: frontend/.env
Comment (Ando22) [reply frontend/.env:1]: oke on it

**Review by polymahh — APPROVED** [review]
seems good to me

**Review by salahlalami — APPROVED** [review]

**Comment by Ando22** [comment]
@polymahh @salahlalami thank you guys, but i will merge this PR after this PR is merged

https://github.com/idurar/erp-crm/pull/552

**Pull request merged**
